spec/db_cleaner_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'fileutils'
+
+RSpec.describe Geordi::DBCleaner do
+  describe '#new' do
+    let(:dir) { File.join(Dir.pwd, 'spec') }
+
+    before do
+      stub_const('ENV', ENV.to_hash.merge('XDG_CONFIG_HOME' => dir))
+    end
+
+    after do
+      file_dir = File.join(dir, '/.config')
+      FileUtils.rm_rf(file_dir)
+    end
+
+    it 'moves the legacy allowlists from ~/.config/geordi/whitelists to ~/.config/geordi/allowlists' do
```

Comment:
Normally, we would have written these specs as Cucumber Aruba features. There is really nice command line support, also for working with files. But it's really nice you got this working in RSpec as well. Let's keep it.

Reply:
I prefer writing rspec to cucumber :sweat_smile: 
For testing commands as a whole cucumber certainly makes sense, but this seemed more like something I should test in a unit test, as it diverges from the "happy path"  imo.